src/components/Navbar/Drawer/Drawer.module.scss

```diff
@@ -44,7 +44,10 @@
 
 .bodyContainer {
   margin-top: constants.$navbar-height;
-  padding: var(--spacing-small);
+  padding-top: var(--spacing-small);
+  padding-bottom: calc(2 * var(--spacing-large));
+  padding-left: var(--spacing-small);
+  padding-right: var(--spacing-small);
```

Comment:
using -top, -bottom, -left, and -right is more verbose. I find it be more clear instead using one liner `paddding: var(--spacing-small) var(--spacing-small) calc(2 * var(--spacing-large)) var(--spacing-small)`;